### PR TITLE
Binary: Make sure we bring orig with us in the multidist tarball

### PR DESCRIPTION
**Pull request by mariogrip**

### CODE REVIEW

File: build-binary.sh [hunk 1]
```diff
@@ -65,7 +65,12 @@ if [ -f multidist.buildinfo ]; then
 	tar -xvzf multidist.tar.gz
 	rm multidist.tar.gz
 	export rootwp=$(pwd)
-
+
```
Comment (peat-psuwit) [inline build-binary.sh]: You left a spurious tab here.

```suggestion

```

File: build-binary.sh [hunk 2]
```diff
@@ -65,7 +65,12 @@ if [ -f multidist.buildinfo ]; then
 	tar -xvzf multidist.tar.gz
 	rm multidist.tar.gz
 	export rootwp=$(pwd)
-
+	
+	# Move orig to mbuild folder
+	find "$rootwp" \
+        -maxdepth 1 -type f -name '*.orig.*' \
+        -exec mv '{}' "$rootwp/mbuild" ';'
+
```
Comment (peat-psuwit) [inline build-binary.sh]: And here.

```suggestion

```

File: build-binary.sh [hunk 3]
```diff
@@ -82,7 +87,7 @@ if [ -f multidist.buildinfo ]; then
 		fi
 
 		# Ensure orig tarball exists
-		find "$rootwp" \
+		find "$rootwp/mbuild" \
```
Comment (peat-psuwit) [inline build-binary.sh]: Maybe use `../` instead, so that the link stays valid when moving between machines.

```suggestion
		find ../ \
```

**Review by peat-psuwit — CHANGES_REQUESTED** [review]
Yeah, we have to carry the orig file to `build-repo.sh` too. However, there're still a bit of a problem.

File: build-binary.sh (hunk 3 above)
Comment (mariogrip) [reply build-binary.sh]: Done

File: build-binary.sh (hunk 1 above)
Comment (mariogrip) [reply build-binary.sh]: blame githubs editor xP

**Review by peat-psuwit — APPROVED** [review]
LGTM now.

**Pull request merged**